### DISCUSSION TITLE
How can I make my Discord bot receive DMs?

### DISCUSSION THREAD

**Discussion by vedantroy** [discordjs/discord.js#9559] [Q&A] [answered]
I have the following very simple bot:
```javascript
const Discord = require('discord.js');

const { User, Message, GuildMember, ThreadMember } = Discord.Partials;
const { Guilds, GuildMessages, DirectMessages } = Discord.GatewayIntents;

const client = new Discord.Client({
  intents: [Guilds, GuildMessages, DirectMessages],
  partials: [User, Message, GuildMember, ThreadMember]
});

client.on('ready', () => {
  console.log(`Logged in as ${client.user.tag}!`);
});

client.on('messageCreate', (message) => {
  if (message.author.bot) return; // Ignore messages from bots
  console.log(`Message received`)

  message.channel.send('Hello');
});

client.login('<BOT_CODE>');
```

I added the bot to my server--and it successfully prints `Message received` to the console whenever someone sends a message in the server. However, I want people to be able to directly DM the bot. Additionally, when people join the server, I'd like the bot to directly DM them.

How can I do this? I thought I had the correct permissions, but the bot is not printing `Message received` when people DM it directly.

I have the `MESSAGE CONTENT INTENT` enabled in Discord's application settings.

**Comment by Idris1401** [comment, marked as answer]
add the `Channel` partial

**Comment by jaw0r3k** [comment]
1. you even don't need message content intent
2. even if you could need it, you would have to set it in the constructor
3. solution is what idris said
